src/views/Connector/Details/Form/connector-utils.js

```diff
@@ -68,6 +68,7 @@ export const getUsersTableHeader = (handleToggleUserRoleClick, handleDeleteUserC
         order: 5,
         modifier: (row) => {
             const isAdmin = row.role === 'admin';
+            const isSelf = row.user === userId;
```

Comment:
I don't think we should do this, as this action can be done through select input